examples/tuners/random_nas_tuner/test.py

```diff
@@ -0,0 +1,82 @@
+                                                                             'in_ch=self.channel_1_num, '
+                                                                             'out_ch=self.channel_2_num)',
+                                                                             'op.dilated_conv(in_ch=self.channel_1_num, '
+                                                                             'out_ch=self.channel_2_num)',
+                                                                             'op.separable_conv(size=3, '
+                                                                             'in_ch=self.channel_1_num, '
+                                                                             'out_ch=self.channel_2_num)',
+                                                                             'op.separable_conv(size=5, '
+                                                                             'in_ch=self.channel_1_num, '
+                                                                             'out_ch=self.channel_2_num)',
+                                                                             'op.separable_conv(size=7, '
+                                                                             'in_ch=self.channel_1_num, '
+                                                                             'out_ch=self.channel_2_num)'],
+                                                            'optional_input_size': [0,
+                                                                                    1],
+                                                            'optional_inputs': ['post1_out']},
+                                        'mutable_layer_4': {'layer_choice': ['op.post_process(ch_size=self.channel_2_num)'],
+                                                            'optional_input_size': 0,
+                                                            'optional_inputs': []},
+                                        'mutable_layer_5': {'layer_choice': ['op.max_pool(size=3)',
+                                                                             'op.max_pool(size=5)',
+                                                                             'op.max_pool(size=7)',
+                                                                             'op.avg_pool(size=3)',
+                                                                             'op.avg_pool(size=5)',
+                                                                             'op.avg_pool(size=7)'],
+                                                            'optional_input_size': [0,
+                                                                                    1],
+                                                            'optional_inputs': ['post1_out',
+                                                                                'pool1_out']}}}})
```

Comment:
This looks strange.
Please define the search space as an explicit variable, with JSON indentation.

Reply:
Looks like I've written this file to examine how this tuner works. Forget to delete it. Removing it now.

Reply:
I thought is was unit test...

Reply:
Seems nice to have a UT for each tuner, but also demands lots of work...

Maybe having a "coverage" badge on our front page will encourage people to write tests... or not.

Reply:
@ultmaster it would be better if you can write unit test